pkg/apis/k0s.k0sproject.io/v1beta1/calico.go

```diff
@@ -48,6 +48,9 @@ type Calico struct {
 
 	// Windows Nodes (default: false)
 	WithWindowsNodes bool `json:"withWindowsNodes"`
+
+	// PrometheusMetricsEnabled enables the Prometheus metrics server in Felix if set to true
+	PrometheusMetricsEnabled bool `json:"prometheusMetricsEnabled"`
```

Comment:
I think, the manifests are required to be regenerated by calling `make manifests`